images/kindnetd/Dockerfile

```diff
@@ -14,10 +14,10 @@
 
 # first stage build kindnetd binary
 # NOTE: tentatively follow upstream kubernetes go version based on k8s in go.mod
-FROM golang:1.13
+FROM golang:1.15
```

Comment:
this should only be bumped if the k8s.io imports match and are using 1.15

Reply:
```
	k8s.io/api v0.19.2
	k8s.io/apimachinery v0.19.2
	k8s.io/client-go v0.19.2
```